pages/api/prescriptions/read.ts

```diff
@@ -9,9 +9,15 @@ export default async function handler(
   // retrieve prescriptions by id, patientId, or don't specify body to retrieve all
```

Comment:
It may be worth making this a `GET` endpoint as per my previous comments. refer to the [next.js API documentation](https://nextjs.org/docs/api-routes/dynamic-api-routes)

Reply:
Refer to later comment about potentially re-structuring endpoints